action/audit.go

```diff
@@ -122,7 +123,7 @@ func (s *Action) audit(validator *crunchy.Validator, secrets <-chan string, chec
 	for secret := range secrets {
 		content, err := s.Store.GetFirstLine(secret)
 		if err != nil {
-			checked <- auditedSecret{name: secret, content: string(content), err: err}
+			checked <- auditedSecret{name: secret, content: string(content), err: err, message: err.Error()}
 			continue
 		}
 
```

Comment:
An error in this case is an error in the pipeline. You're now adding  messages as errors too.  I'm not really sure that this actually does what it is supposed to do. 
Will check this tomorrow.